src/instructlab/model/evaluate.py

```diff
@@ -333,6 +333,12 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--background",
+    type=click.BOOL,
+    default=False,
+    help="Run vLLM in the background.",
```

Comment:
I don't think this should be vLLM specific, maybe it's a wording issue and we can make this backend agnostic or reflect it's only implemented for a specific backend only.

Reply:
++ should be for llama.cpp as well.